BlogTemplate/Pages/Account/Login.cshtml

```diff
@@ -45,9 +45,12 @@ else
                         <p>

```

Comment:
This is just nitpicking, but in line 12, the sentence "You cannot register because you are already logged in." doesn't really make sense for the login page. Maybe change it to just "You are already logged in."

Reply:
This has been fixed.

Reply:
@lucasisaza When I go to /account/login when I'm already logged in, it still shows this page with the weird sentence. Not sure if this is what you intended it to do.

Reply:
Good find, I just added it to the new commit.